packages/components/organisms/f-checkout/src/components/Guest.vue

```diff
@@ -103,7 +113,11 @@ export default {
     methods: {
         ...mapActions(VUEX_CHECKOUT_MODULE, [
             'updateCustomerDetails'
-        ])
+        ]),
+
+        isFieldValid (field) {
```

Comment:
I would expect a method with this name to return a boolean, but it doesn't look like it does that. Is this method needed?